base/Package.md

```diff
@@ -475,7 +474,3 @@ This package supports all the data types defined in specification [Data Types](h
 [s143]: http://hl7.org/fhir/StructureDefinition/MedicinalProductAuthorization
 [s144]: http://hl7.org/fhir/StructureDefinition/Claim
 [s145]: http://hl7.org/fhir/StructureDefinition/Location
-
-[mod1]: https://lib.ballerina.io/ballerinax/health.fhir.r4.parser/1.0.3
-[mod2]: https://lib.ballerina.io/ballerinax/health.fhir.r4.terminology/1.0.3
-[mod3]: https://lib.ballerina.io/ballerinax/health.fhir.r4.validator/1.0.3
```

Comment:
These links are referred to in the Submodules section.

Reply:
Seems these are moved to lines https://github.com/ballerina-platform/module-ballerinax-health.fhir.r4/pull/25/files#diff-60dfdceaf7f423f59e8dfaf19962afa93424b3aa220d80809bffe41445e49c1fR182